emperor/support_files/js/view.js

```diff
@@ -41,6 +41,13 @@ function DecompositionView(decomp) {
    */
   this.visibleDimensions = [0, 1, 2]; // We default to the first three PCs
   /**
+   * Orientation of the axes, `-1` means the axis is flipped, `1` means the
+   * axis is not flipped.
+   * @type {integer[]}
+   * @default [1, 1, 1]
+   */
+  this.axesOrientation = [1, 1, 1];
```

Comment:
Probably I'm missing something, but can this be a boolean? looking at the code in here it doesn't look like you're doing any computation with them.